docling/backend/abstract_backend.py

```diff
@@ -38,6 +38,10 @@ def unload(self):
     def supported_formats(cls) -> Set["InputFormat"]:
         pass
 
+    @abstractmethod
```

Comment:
This @abstractmethod tag should not be necessary since you are providing a default implementation.